app/services/prison_visits/client.rb

```diff
@@ -1,4 +1,5 @@
 require 'excon'
+require 'rack/utils'
```

Comment:
Do we need this `require`?

Reply:
Always good for a class to specify its dependency. Potentially this file doesn't require Rails to work. Excon is required as well I assume for the same reason. This file also depend on `request_store` and `json` which could also be required.

Ultimately all NOMIS related stuff could be put under `lib/` as most (none maybe?) of it requires Rails to work.

But I'm ok with removing the require(s)

Reply:
But this require was for the other attempted fix on the params that was using rack/utils, the actual fix does not end using it so this should have been removed also